app/registries/menu/loader.rb

```diff
@@ -94,12 +94,7 @@ def self.load
       end
 
       Manager.map :labs_menu do |menu|
```

Comment:
Do we still need this empty menu? Or is it used by some plugins?

Reply:
never mind. looks like it is.

Reply:
it is used by REX for the new job wizard and also by Katello CV if I recall